app/search_builders/curation_concerns/filter_suppressed_with_roles.rb

```diff
@@ -0,0 +1,28 @@
+module CurationConcerns
+  module FilterSuppressedWithRoles
```

Comment:
Add a module comment about when this can be used and its assumptions (`solr_params[:id]` being one).